### PR TITLE
Update quick start for SDK revisions

### PR DESCRIPTION
**Pull request by shamsway**
Updated quick start for SDK revisions, mainly around error handling.

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -120,53 +121,82 @@ package main
...
+	"log"
+
 	"github.com/rubrikinc/rubrik-sdk-for-go/rubrikcdm"
 )
 
 func main() {
+
 	// Establish a connection to the Rubrik cluster
-	rubrik := rubrikcdm.ConnectEnv()
+	rubrik, err := rubrikcdm.ConnectEnv()
```
Comment (chriswahl) [inline docs/quick-start.md]: I get an error on this - `assignment mismatch: 2 variables but 1 values`

Comment (drew-russell) [reply docs/quick-start.md]: @chriswahl Did you upgrade your code to the `devel` branch? That error is consistent with how everything looked before the new updates.

Comment (chriswahl) [reply docs/quick-start.md]: I'm using the `shamsway:master` branch.

Comment (shamsway) [reply docs/quick-start.md]: @chriswahl there isn't a quick start in the `devel` brach, so I forked the master branch to make updates. It makes sense to me to have the quick start in the `devel` branch as well, so we will do that moving forward.

**Comment by drew-russell** [comment]
@chriswahl I'm good with approving this and then merging `devel` to make sure everything is up to date.

**Comment by chriswahl** [comment]
> @chriswahl I'm good with approving this and then merging `devel` to make sure everything is up to date.

Sounds good to me. :shipit:

**Pull request merged**